python/az/aro/azext_aro/custom.py

```diff
@@ -355,8 +355,7 @@ def generate_random_id():
 
 
 def get_network_resources_from_subnets(cli_ctx, subnets, fail):
-    network_client = get_mgmt_service_client(cli_ctx, ResourceType.MGMT_NETWORK)
-
+    from .aaz.latest.network.vnet.subnet import Show
```

Comment:
Same as above. `import`s are best at the top of the file.

Reply:
fixed